helm/default-apps-azure/values.yaml

```diff
@@ -3,21 +3,33 @@ organization: ""
 
 # filled by cluster-apps-operator
 # via <cluster-name>-cluster-values configmap
+baseDomain: ""
 cluster:
   private: false
+subscriptionID: ""
+identityClientID: ""
 
 userConfig:
   certManager:
     configMap:
-      values:
-        # cert-manager's DNS01 solver by default tries to reach authoritative nameservers directly, using
-        # their public IPs. Since those are not reachable from private clusters, we instead rely on the
-        # recursive nameserver
-        #
-        # For public clusters, this setting should have no effect, as they can use the HTTP01 solver.
+      values: |
```

Comment:
@fiunchinho When I use map instead of string here, it is not possible to template values conditionally. Do you have an idea about how we can do it?

Reply:
Jose is on paternal leave, but we should not go back to the old way of configuring via string, as this would bring inconsistency.
 
 But I do not know the solution

Reply:
@calvix I used extra-config to solve the templating problem. Are you OK with it?